org.eclipse.jdt.ls.tests.syntaxserver/src/org/eclipse/jdt/ls/core/internal/syntaxserver/SyntaxServerTest.java

```diff
@@ -262,6 +262,17 @@ public void testHoverUnresolvedType() throws Exception {
 		assertEquals("This is interface IFoo.", list.get(1).getLeft());
 	}
 
+	// https://github.com/redhat-developer/vscode-java/issues/1931
+	@Test
+	public void testParameterMismatch() throws Exception {
+		openFile("maven/salut4", "src/main/java/pack/TestSyntax.java");
```

Comment:
Looks like TestSyntax is missing from the PR.

Reply:
Fixed.